source/community/release_notes/quince.rst

```diff
@@ -27,14 +31,36 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+
+Which MFEs will be included?
+
+* `frontend-app-learner-record: <https://github.com/openedx/frontend-app-learner-record>`__
+* `frontend-app-learner-dashboard: <https://github.com/openedx/frontend-app-learner-dashboard>`__
+
+New Waffle Flagts:
+
+* `feat: add Waffle Flag to disable resetting self-paced deadlines by learners: <https://github.com/openedx/edx-platform/pull/32148>`__
+* `feat: adds waffle flag for show notifications tray: <https://github.com/openedx/edx-platform/pull/32451>`__
+* `feat: add course waffle flag for learner assistant: <https://github.com/openedx/edx-platform/pull/32657>`__
 
 Deprecations & Removals
```

Comment:
There's a list of deprecations at https://openedx.atlassian.net/wiki/spaces/COMM/pages/3726802953/Pending+Release+Quince#Deprecations-and-Removals-%E2%9C%96%EF%B8%8F -- you should check to see that everything listed there is covered here.